source/h2_stream.c

```diff
@@ -1254,6 +1254,14 @@ static int s_stream_write_data(
     struct aws_http_stream *stream_base,
     const struct aws_http2_stream_write_data_options *options) {
     struct aws_h2_stream *stream = AWS_CONTAINER_OF(stream_base, struct aws_h2_stream, base);
+    if (!stream->manual_write) {
+        AWS_LOGF_ERROR(
+            AWS_LS_HTTP_STREAM,
+            "Manual writes are not enabled. You need to enable manual writes using by setting "
+            "`http2_use_manual_data_writes` to true in `aws_http_make_request_options`, stream=%p",
```

Comment:
trivial: I have no idea if there's anything wrong with backticks in a log message, but it is unusual? single-quote is more typical

Also, use `AWS_H2_STREAM_LOGF()` so we format our log messages consistently from this file. Looks like a few vanilla `AWS_LOGF_ERROR()` slipped into this file; they should probably be changed too

Reply:
Thanks, updated.